packages/fx-core/tests/plugins/resource/bot/unit/expectedBicepFiles/botProvisionOfFuncHosted.result.bicep

```diff
@@ -70,7 +70,7 @@ resource functionApp 'Microsoft.Web/sites@2021-02-01' = {
         }
         {
           name: 'FUNCTIONS_EXTENSION_VERSION'
-          value: '~3' // Use Azure Functions runtime v3
+          value: '~4' // Use Azure Functions runtime v4
```

Comment:
Seems also need to set `netFrameworkVersion` to `v6.0` because we use Windows Functions App.
https://docs.microsoft.com/en-us/azure/azure-functions/functions-versions?tabs=in-process%2Cazure-cli%2Cv4&pivots=programming-language-javascript#azure

> For Windows function apps only, enable .NET 6.0 through the netFrameworkVersion setting

Reply:
Is `netFrameworkVersion` required even we set `FUNCTIONS_WORKER_RUNTIME` to *node*?

Reply:
From my understanding of the doc, yes. And I am testing not settings `netFrameworkVersion`. The current result is the HTTP status code returns 200 but never receives notification.

Reply:
I just created a Windows function app through Azure Portal (node.js 16, function runtime 4). There is not `netFrameworkVersion` setting in my function app's configuration. 
I think the document means function runtime 4 only supports .NET 6.0 so for those .NET runtime function app, they should upgrade the .NET version. We do not need this setting in node.js runtime function.

Reply:
Tested again with Function app v4 and without settings `netFrameworkVersion`. I can successfully receive the notification. Previous failure to receive notification may be due to app studio error (Something wrong with app studio, the teams app creation succeeded but I cannot find it in the portal).